app/src/androidTest/java/com/github/se/project/ui/profile/ProfileInfoScreenTest.kt

```diff
@@ -0,0 +1,210 @@
+package com.github.se.project.ui.profile
```

Comment:
By adding the file here do we still need to merge the other PR?

Reply:
I thinks it is good if we merge the other PR before this one so that we know from which PR the work come from.